mcs/class/System/System.IO.Compression/DeflateStream.cs

```diff
@@ -611,6 +611,7 @@ static int WriteZStream (SafeDeflateStreamHandle stream, IntPtr buffer, int leng
 			}
 		}
 #else
+		// FIXME? Names should start "mono"?
```

Comment:
I don't think this is worth fixing as I've seen external user code with a `DllImport ("MonoPosixHelper", ..)` that would be broken by this.

Reply:
Yeah, it is a tough situation, agreed.

Because, now (and forever already) mono can end up replacing system libz.so.
You know, if it loads first, on most Unix systems, with their flat namespace.
So the responsibility to export a very good version is high.
I think.

I can just remove the comments then.

Reply:
All of the names?

Reply:
i.e. just the zlib ones or also the helper ones?